lesson_06/homework_6.1.py

```diff
@@ -0,0 +1,24 @@
+"""Module providing a new function printing python version."""
+
+
+
+print("Enter your sentence")
+uniq_symbols = input()
+for c in uniq_symbols:
+    if len(set(uniq_symbols.lower())) >= 10:
+        print(True)
+    else:
+        print(False)
+    break
```

Comment:
Here for cycle not needed - we check len of input .
No need to iterate over each character